profiles/matthewraymondh.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/83885819/147108033-04ee7975-97ac-421c-96cc-7330603ed1dd.png",
```

Comment:
I recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/147137654-7f453c7e-b6f5-4b68-bcfc-fd9c0a104c04.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/147137654-7f453c7e-b6f5-4b68-bcfc-fd9c0a104c04.png",
```